transcom/mymove
# B-18986 Customer update UI to show the shipment locator

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18986)

## AC for this story
Given I'm the customer
When reviewing my move during onboarding then I will see the shipment IDs on the "Review your details" page, presented with each shipment under the "move setup" section.

Given I'm the customer
When viewing my current move, after signing in and navigating to "go to move" then I will see the Shipment ID next to my shipments in the "Shipments" section.

***Note: in the AC they are referencing Shipment ID or Shipment IDs but they mean the newly created locator called Shipment Locator***

## Testing Steps I took
1. Since creating a move is not necessary, you can just log in with any use that has at least 1 shipment on a move.
2. Click on the Go to Move button and you should see the shipments on that move with the shipment locator for that shipment on the same row (see Fig 1).
3. To verify that the information presented is correctly pulling from the DB, run the following SQL query (change out 6DHY4D for your move locator):
```
select ms.shipment_locator from mto_shipments ms join moves m on m.id = ms.move_id where m."locator" = '6DHY4D';
```
4. You should see the same number of results as you see on the frontend site.

# Fig 1
![Screenshot 2024-04-11 at 3 42 23 PM](https://github.com/transcom/mymove/assets/147739091/04969949-9bc1-48d6-a9f6-f509fc5f339b)
